Exersises/test1.cpp

```diff
@@ -0,0 +1,37 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
+#include <iostream>
+#include <vector>
+#include "1.h"
+#include "doctest.h"
+
+using namespace std;
+
+TEST_CASE("simple test") {
+    vector<pair<int, int>> edges = {
+        {0, 1}, {1, 2}, {2, 3}, {3, 0}
+    };
+    vector<pair<int, int>> pairs = {
+        {0, 1}, {2, 3}
+    };
+    disjoint_path_finder q(4, edges, pairs);
```

Comment:
If you pass a constant to a function/constructor, add a comment to show the name of the argument, like so:
disjoint_path_finder(/*n=*/, ...)